frontend/src/components/text/Text.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useRef, useState } from "react";
+
+interface Props {
+  displayedText: string;
+  delay?: number;
+}
+
+const Text = ({ displayedText, delay = 100 }: Props) => {
+  const [textCount, setTextCount] = useState(0);
```

Comment:
```suggestion
  const [textCounter, setTextCounter] = useState(0);
```
textCount stands more for "liczenie tekstu", not "licznik tekstu/wyrazów", if you make a function out of this, then countText would be proper